server/presenters/collection.js

```diff
@@ -40,14 +39,6 @@ async function present(ctx: Object, collection: Collection) {
     data.documents = sortDocuments(collection.documentStructure);
   }
 
-  if (collection.documents) {
-    data.recentDocuments = await Promise.all(
```

Comment:
This key was never used, and not documented so removing for the performance boost as we're adding more queries to loading collections now.